keep/api/models/alert.py

```diff
@@ -385,6 +385,8 @@ class IncidentDto(IncidentDtoIn):
 
     rule_fingerprint: str | None
 
+    _tenant_id: str = PrivateAttr()
```

Comment:
i admit i'm not fully sure i understand this one, it's a bit complex, why isn't `alerts` just a function instead of a property?